website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -87,6 +87,21 @@ Nomad no longer creates an implicit Consul identity for workloads that don't
 register services with Consul. Tasks that require Consul tokens for template
 rendering must include a [`consul` block][] or specify an [`identity`][].
 
+#### OIDC login with PKCE
+Nomad now enables [PKCE](https://oauth.net/2/pkce/) (Proof Key for Code
+Exchange) by default for new or updated OIDC auth methods. Existing auth
+methods remain unaffected until changed using the
+[acl auth-method create](/nomad/docs/commands/acl/auth-method/create) or
+[acl auth-method update](/nomad/docs/commands/acl/auth-method/update)
+CLI commands, or with the
+[auth method API](/nomad/api-docs/acl/auth-methods) directly.
+
+The [`OIDCDisablePKCE`](/nomad/api-docs/acl/auth-methods#oidcdisablepkce)
+option may be set to turn off this extra security.
+
+Note that although Nomad enables PKCE by default, some OIDC providers may
+require you to also enable it in their configuration.
+
```

Comment:
I know we often prefer `[footer links][]`, but on this particular page, which has loooots of links relating to all kinds of things, it felt better to keep all the context together here.